index.js

```diff
@@ -104,6 +105,46 @@ export async function registerSpace (email) {
   console.log(`⁂ space registered to ${email}`)
 }
 
+/**
+ * @param {string} proofPath
+ */
+export async function addSpace (proofPath) {
+  const client = await getClient()
+  const reader = await CarReader.fromIterable(fs.createReadStream(proofPath))
+  const blocks = []
+  for await (const block of reader.blocks()) {
+    blocks.push(block)
+  }
+  // @ts-expect-error
+  const delegation = importDAG(blocks)
+  const space = await client.addSpace(delegation)
+  console.log(space.did())
+}
+
+export async function listSpaces () {
+  const client = await getClient()
+  const current = client.currentSpace()
+  for (const space of client.spaces()) {
+    const prefix = current && current.did() === space.did() ? '* ' : '  '
+    console.log(`${prefix}${space.did()} ${space.name() ?? ''}`)
+  }
+}
+
+/**
+ * @param {string} did
+ */
+export async function useSpace (did) {
```

Comment:
musing: maybe a prompt version of this _would_ be nice... where you can use the arrow keys to iterate through the options and then hit return

Reply:
yes maybe if no arg?